cylc/flow/scheduler_cli.py

```diff
@@ -107,6 +108,167 @@
 }
 '''
 
+PLAY_OPTIONS = [
+    {
+        ARGS: ["-n", "--no-detach", "--non-daemon"],
+        KWARGS: {
+            HELP: "Do not daemonize the scheduler (infers --format=plain)",
+            ACTION: "store_true",
+            DEST: "no_detach",
+        }
+    },
+    {
+        ARGS: ["--profile"],
+        KWARGS: {
+            HELP: "Output profiling (performance) information",
+            ACTION: "store_true",
+            DEFAULT: False,
+            DEST: "profile_mode"
+        }
+    },
+    {
+        ARGS: ["--start-cycle-point", "--startcp"],
+        KWARGS: {
+            HELP:
+                "Set the start cycle point, which may be after "
+                "initial cycle point. If the specified start point "
+                "not in the sequence, the next on-sequence point "
+                "be used. (Not to be confused with the initial ",
+            METAVAR: "CYCLE_POINT",
+            ACTION: "store",
+            DEST: "startcp",
+        }
+    },
+    {
+        ARGS: ["--final-cycle-point", "--fcp"],
+        KWARGS: {
+            HELP:
+                "Set the final cycle point. This command line option "
+                "the workflow config option '[scheduling]final ",
+            METAVAR: "CYCLE_POINT",
+            ACTION: "store",
+            DEST: "fcp",
+        }
+    },
+    {
+        ARGS: ["--stop-cycle-point", "--stopcp"],
+        KWARGS: {
+            HELP:
+                "Set the stop cycle point. Shut down after all "
+                "have PASSED this cycle point. (Not to be confused "
+                "the final cycle point.) This command line option "
+                "the workflow config option '[scheduling]stop after ",
+            METAVAR: "CYCLE_POINT",
+            ACTION: "store",
+            DEST: "stopcp",
+        }
+    },
+    {
+        ARGS: ["--start-task", "--starttask", "-t"],
+        KWARGS: {
+            HELP:
+                "Start from this task instance, given by '<cycle>/<name>'. "
+                "be used multiple times to start from multiple "
+                "at once. Dependence on tasks with with cycle points "
+                "than the earliest start-task will be ignored. "
+                "sub-graph of the workflow will run if selected ",
```

Comment:
I think this needs checking. There is some deviation from the original text which is a bit confusing.
```suggestion
                "Start from this task instance, given by '<cycle>/<name>'. "
                "This can be used multiple times to start from multiple "
                "tasks at once. Dependence on tasks with cycle points earlier"
                "than the earliest start-task will be ignored. A"
                "sub-graph of the workflow will run if selected tasks do not lead on to the full graph.",
```